src/main.cpp

```diff
@@ -673,33 +745,23 @@ int main(int argc, char* argv[])
   }
 
   // Listing probes
-  if (listing)
+  if (args.listing)
   {
     if (!is_root())
       return 1;
 
-    if (optind == argc || std::string(argv[optind]) == "*")
-      script = "*:*";
-    else if (optind == argc - 1)
-      script = argv[optind];
-    else
-    {
-      usage();
-      return 1;
-    }
```

Comment:
Moved into parse_args, as it deals with processing a positional parameter. See `args.search`